### ISSUE TITLE
Tab Order dialog/overlay missing in Core Designer

### ISSUE THREAD

**Issue by Dave-Lowndes** [dotnet/winforms#3509]
**Is your feature request related to a problem? Please describe.**
    The method of setting control keyboard (tab) ordering for WinForms is poor in comparison to the Windows resource dialog editor. The dialog editor allows you to Ctrl+click a control to start tab re-ordering at that point. I'm not aware of a similar facility for WinForms, consequently it's always an irritation when you need to re-do tab ordering.

**Describe the solution you'd like and alternatives you've considered**
    If you can do it like the dialog editor (or better) that'd be great.

**Will this feature affect UI controls?**
    No

**Comment by weltkante** [comment]
on the "layout" toolbar you can open the dropdown add/remove buttons and show the "Tab Order" command. This lets you enter a mode where you can sequentially click controls to determine their TabOrder property.

This only works in Desktop Framework Designer currently, .NET Core Designer seems to ignore it. Also its not really discoverable if you don't know about it.

**Comment by Dave-Lowndes** [comment]
@weltkante Hi Tobias, as far as I can see, that button is the same as the View, Tab Order menu item. It doesn't provide any additional functionality. The Windows dialog editor let's you Ctrl+click to start ordering at the current index of the control you click - it's a really useful facility when you don't need to re-do from 0 a form that has many controls on it.

**Comment by merriemcgaw** [comment]
We're working on this for an upcoming release. I totally agree that it's quite inconvenient without it.

**Comment by jnm2** [comment]
Is this issue tracking the enabling of the tab order IDE function in the .NET Core/5 designer, or specifically only the Ctrl+click part?

**Comment by JeremyKuhne** [comment]
@jnm2 There is a designer issue tracking getting the existing functionality working again. I've made sure the links are in place. We're sort of piggy-backing the two things in this public issue.

**Comment by RussKie** [comment]
FYI, it is being worked on. https://twitter.com/IgorRussKie/status/1467788326493061120

**Comment by Dave-Lowndes** [comment]
@Shyam-Gupta
In which version of VS should we see this improved facility? I've just updated VS2022 to 17.1.0 Preview 2.0, and for a new WinForm project targeting .Net Core 6, the View, Tab Order menu item seems to be permanently disabled.

**Comment by RussKie** [comment]
This will ship in 17.1 GA
